libs/agno/agno/workflow/v2/step.py

```diff
@@ -211,6 +207,12 @@ def execute(
 
                     # Execute agent or team with media
                     if self._executor_type in ["agent", "team"]:
+                        # Switch to appropriate logger based on executor type
+                        if self._executor_type == "agent":
```

Comment:
is this absolutely needed?
i think when the executor is agent, the logs automatically are coming from agent then right... so its automatically switched with the executor, shouldn't be needed here. But lemme know if i'm misunderstanding it.